athena_cfn/README.md

```diff
@@ -0,0 +1,180 @@
+# Query Quilt Package Metadata in AWS Athena
+
+The code in this example shows you how to enable the "Queries"
+tab of the Quilt Catalog for querying Quilt [metadata](https://docs.quiltdata.com/catalog/metadata)
+in SQL with [Quilt's Athena Integration](https://docs.quiltdata.com/advanced/athena).
+
+## I. Prologue: Ensure Source=Quilt Roles for Athena users
+
+Quilt stacks ship with two `Source=Custom` roles that
+automatically grant access to all buckets that have been added to Quilt.
+At this time `Source=Custom` Roles are incompatible with custom policies
+(including Athena policies). If you have not already,
+before proceeding you must do the following in the Quilt Admin panel:
+
+1. Create a new Quilt Role (`Source=Quilt`), such as "AthenaQuiltAccess"
+1. Create a new Quilt Policy (`Source=Quilt`) with access to any
```

Comment:
Ah, now I know why this is confusing. We need to explain way up front that `Source=Quilt` means check the box "Manually set ARN instead of configuring policies" and provide your own policy ARN.

Reply:
Um, I think that is the other way around.  `Source=Custom` means manually-set ARNs, `Source=Quilt` means configured directly from the UI.

I agree that https://docs.quiltdata.com/catalog/admin needs to be MUCH clearer on this, but I wanted to decouple that from this PR.